sdk/monitor/monitor-opentelemetry-exporter/src/generated/models/index.ts

```diff
@@ -23,7 +23,7 @@ export interface TelemetryItem {
   /**
    * Event date time when telemetry item was created. This is the wall clock time on the client when the event was generated. There is no guarantee that the client's time is accurate. This field must be formatted in UTC ISO 8601 format, with a trailing 'Z' character, as described publicly on https://en.wikipedia.org/wiki/ISO_8601#UTC. Note: the number of decimal seconds digits provided are variable (and unspecified). Consumers should handle this, i.e. managed code consumers should not use format 'O' for parsing as it specifies a fixed length. Example: 2009-06-15T13:45:30.0000000Z.
    */
-  time: Date;
+  time: string;
```

Comment:
did we really mean to make this a string and not a Date? Usually Date is easier to work with.

Reply:
Agreed. Have you gotten dates to be cross-lang interoperable in other client SDKs? Maybe the mapper/serializer could handle mapping Date to string?

Reply:
Doesn't it work if you do something like this?
```
"startTime": {
  "type": "string",
  "format": "date-time",
  "description": "The time when an indexer should start running."
}
```